tests/test_importers.py

```diff
@@ -6,13 +6,15 @@
 import os
 
 from cadquery import *
-from cadquery import exporters
 from cadquery import importers
```

Comment:
Doesn't the `from cadquery import *` line above make this redundant?

Reply:
Corrected